src/general/CombinableActionData.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+public abstract class CombinableActionData
+{
+    /// <summary>
+    ///   Does this action cancel out with the <paramref name="other"/> action?
+    /// </summary>
+    /// <returns>
+    ///   Returns the interference mode with <paramref name="other"/>
+    /// </returns>
+    /// <para>Do not call with itself</para>
```

Comment:
Needs to be wrapped in remarks. Also I'd still suggest you implement a body for this method here that checks that this isn't called with self and then calls a protected abstract method.